packages/vue/__tests__/Transition.spec.ts

```diff
@@ -59,7 +59,6 @@ describe('e2e: Transition', () => {
         // leave
         expect(await classWhenTransitionStart()).toStrictEqual([
           'test',
-          'v-leave-active',
```

Comment:
These existing test cases should not change. That behavior is correct

Reply:
I've changed some of transform logic at `Transition.ts` , so that the test cases should follow the changes. I'm going to analyze it a little bit that why in this way.

Reply:
What I mean is that the fix should not require to change those tests